promptsource/templates/lama/google_re/templates.yaml

```diff
@@ -0,0 +1,80 @@
+dataset: lama
+subset: google_re
+templates:
+  89ba7edc-d980-4431-8f82-9c07d28d1089: !Template
+    answer_choices: null
+    id: 89ba7edc-d980-4431-8f82-9c07d28d1089
+    jinja: 'Given the following sentence: {{masked_sentence | replace(''[MASK]'',
+      obj_label) }} Rewrite the following template with replacing X and Y with the
+      correct words. {{template}} |||  {{template | replace(''[X]'', sub_label) |
+      replace(''[Y]'', obj_label)}}'
+    metadata: !TemplateMetadata
+      choices_in_prompt: false
+      metrics:
+      - Accuracy
+      - BLEU
+      original_task: false
```

Comment:
We're 1 short on original tasks - we need 5 original tasks per subtask

Reply:
Fixed. Now, this subset has six prompts, five of which are original tasks. Thank you